aten/src/ATen/native/cuda/LinearAlgebra.cu

```diff
@@ -72,27 +49,34 @@ Tensor prepare_matrix_for_cublas(Tensor& tensor, bool& transpose_tensor) {
   return tensor_;
 }
 
+namespace {
+
 Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& mat1, const Tensor& mat2, Scalar beta, Scalar alpha) {
```

Comment:
why do we still have `addmm` in TH (`aten/src/TH/generic/THTensorMath.cpp`)

Reply:
`static void THTensor_(addmmImpl)(THTensor *r_, THTensor *t, THTensor *m1, THTensor *m2, scalar_t beta, scalar_t alpha)`

Reply:
The scope of this PR is only the CUDA part. (THC -> ATen/native/cuda)

Reply:
oh okay!